shell.nix

```diff
@@ -1,21 +1,11 @@
-{ nixpkgs ? import ./nix/nixpkgs.nix {} }:
-
-with nixpkgs;
-
-mkShell {
-  name = "clash-compiler-shell";
-  buildInputs = [
-    # For nix dependency management
-    niv
-
-    # For quick clash experimentation
-    (pkgs.haskellPackages.ghcWithPackages (p: with p; [
-      clash-ghc
-
-      ghc-typelits-extra
-      ghc-typelits-knownnat
-      ghc-typelits-natnormalise
-    ])
-    )
-  ];
-}
+(import
```

Comment:
Is this necessary? I never have `shell.nix` files with flakes

Reply:
Just saw your comment:

> Backwards compatibility is provided via flake-compat so non-flake tooling can still integrate via shimmed default.nix and shell.nix

What tooling are you thinking of? Anything QBL uses or rather external users?